src/components/base/containers/Drawer.js

```diff
@@ -138,4 +138,6 @@ Drawer.defaultProps = {
   openedIconName: 'minus'
 };
 
+Drawer.Panel = DrawerPanel;
```

Comment:
I love namespaced components. Should we also update the documentation for `children` above to use `Drawer.Panel` instead of `DrawerPanel`?